va/va_enc_av1.h

```diff
@@ -0,0 +1,974 @@
+     *
+     * The bitrate can be modified later on through
+     * #VAEncMiscParameterRateControl buffers.
+     */
+    uint32_t    bits_per_second;
+
+    union {
+        struct {
+            /** \brief Still picture encoding, no inter frame referencing. */
+            uint32_t    still_picture                               : 1;
+            /** \brief Force using 128x128 or 64x64 Supper block */
+            uint32_t    use_128x128_superblock                      : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_filter_intra                         : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_intra_edge_filter                    : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_interintra_compound                  : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_masked_compound                      : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_warped_motion                        : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_dual_filter                          : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_order_hint                           : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_jnt_comp                             : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_ref_frame_mvs                        : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_superres                             : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_cdef                                 : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
+            uint32_t    enable_restoration                          : 1;
+            /** \brief Reserved bytes for future use, must be zero. */
+            uint32_t    reserved_bits                               : 18;
+        } bits;
+        uint32_t value;
+    } seq_fields;
+
+    /** \brief Corresponds to AV1 syntax element of the same name.
+     *  value range [0..7].
+     */
+    uint8_t     order_hint_bits_minus_1;
+
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t                va_reserved[VA_PADDING_HIGH];
+} VAEncSequenceParameterBufferAV1;
+
+#define VA_AV1_MAXNUM_SEGMENTS                  8
+#define VA_AV1_MAXNUM_SEGFEATURES               8
```

Comment:
is it SEG_LVL_MAX?

Reply:
yes

Reply:
yes, just to align previous VA_AV1_MAXNUM_SEGMENTS. Agree it may be more clear if changed to SEG_LVL_MAX, which is aligned with spec.